src/icons/index.js

```diff
@@ -9,6 +9,7 @@ export CalendarIcon from './CalendarIcon'
 export CheckIcon from './CheckIcon'
 export CloseIcon from './CloseIcon'
 export CreditIcon from './CreditIcon'
+export DotDotDotIcon from './DotDotDotIcon'
```

Comment:
Love the name, remembers me of this shitty band
![](https://lastfm-img2.akamaized.net/i/u/300x300/7f721cd087664474c462a175b84c5d88.jpg)